python/py-gitfs/Portfile

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-gitfs
+github.setup        presslabs ${python.rootname} 0.5.2
+revision            0
+
+categories-append   devel
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         a FUSE file system that fully integrates with git
+long_description    ${python.rootname} is {*}${description}. You can mount a\
+                    remote repository’s branch locally, and any subsequent\
+                    changes made to the files will be automatically committed\
+                    to the remote.
+
+homepage            https://www.presslabs.com/docs/code/${python.rootname}/
+
+checksums           rmd160  1bdfaec8284ac421fe6e78330ddc118748b599b4 \
+                    sha256  c0bdca63f3fc7cb88ab01d9a5cfd1b345de52ffe9c5cbcb1a27dbe03e0f66b49 \
+                    size    64165
+
+python.versions     39
+
+if {${name} ne ${subport}} {
+    patchfiles              patch-mounter.py.diff \
+                            patch-utils-args.py.diff
+
+    post-patch {
+        reinplace "s|@@PREFIX@@|${prefix}|g" ${worksrcpath}/gitfs/utils/args.py
+    }
+
+    depends_build-append    port:py${python.version}-setuptools
```

Comment:
it [installs](https://github.com/presslabs/gitfs/blob/4a6720ffa00a87fb47a796bb4bb7d4cb9086ae51/setup.py#L15) a `console_script` which means that `py-setuptools` is also needed at runtime, please change the dependency to `depends_lib`